src/pages/index.jsx

```diff
@@ -8,6 +8,12 @@ import useIsBrowser from '@docusaurus/useIsBrowser';
 export default function() {
   const isBrowser = useIsBrowser();
   const {siteConfig} = useDocusaurusContext();
+  const {globalData} = useDocusaurusContext();
+
+  globalData['docusaurus-plugin-content-docs'].default.versions[1].label = '1.1.1';
+  globalData['docusaurus-plugin-content-docs'].default.versions[1].isLast = false;
+  globalData['docusaurus-plugin-content-docs'].default.versions[0].isLast = true;
+
```

Comment:
Each new document version needs to modify this configuration,
Is it possible to automatically identify here.
About shenyu-website, refer  it achieve? 
https://github.com/apache/incubator-shenyu-website/blob/main/docusaurus.config.js

Reply:
Configured from JSON configuration file instead. @casionone